connector/models/checkpoint.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2013-2017 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+"""
+The checkpoint is a model containing records to be reviewed by the end
+users.  The connectors register records to verify so the user can check
+them and flag them as reviewed.
+
+A concrete use case is the import of new products from Magento. Once
+they are imported, the user have to configure things like the supplier,
+so they appears in this list.
+"""
+
+from odoo import _, api, fields, models
+
+
+class ConnectorCheckpoint(models.Model):
```

Comment:
Maybe the good time to do this? https://github.com/OCA/connector/issues/351

Reply:
good idea !

Reply:
Can this be removed as suggested?

Reply:
I think so, nobody is using it AFAIK.
@eantones likely this is the last bit :wink: